reactome-jenkins-release-config-template.properties

```diff
@@ -0,0 +1,54 @@
+releaseNumber=
+
+pathToSpeciesConfig=
+pathToOrthopairs=
+pathToWebsiteFilesUpdateFolder=
+pathToSkipList=
+dateOfRelease=yyyy-mm-dd
+
+pathToSpeciesConfig=
+pantherFileURL=ftp://ftp.pantherdb.org/ortholog/current_release/
+pantherQfOFilename=QfO_Genome_Orthologs.tar.gz
+pantherHCOPFilename=Orthologs_HCOP.tar.gz
+MGIFileURL=http://www.informatics.jax.org/downloads/reports/HGNC_homologene.rpt
+RGDFileURL=ftp://ftp.rgd.mcw.edu/pub/data_release/GENES_RAT.txt
+XenbaseFileURL=ftp://ftp.xenbase.org/pub/GenePageReports/GenePageEnsemblModelMapping.txt
+ZFINFileURL=https://zfin.org/downloads/ensembl_1_to_1.txt
+
+brenda.password=
+orphanet.user=
+cosmic.user=
+brenda.user=
+orphanet.password=
+cosmic.password=
+dockblaster.password=
+dockblaster.user=
+
+absoluteReleaseDirectoryPath=
+releaseDownloadDirectoryPath=
+protegeexporter.pathToWrapperScript=
+# 5 is for an 8-core system (need to leave some resources for MySQL and the OS).
+# If you have a different number of cores, you can change this value, but try to 
+# leave at least 2 or 3 cors available for other processes (such as MySQL and OS). 
+protegeexporter.parallelism=5
+#protegeexporter.extraIncludes=-I/home/ubuntu/perl5/lib/perl5/,-I/home/$USER/perl5/lib/perl5/
```

Comment:
Why is this one commented out?

Reply:
@SolomonShorser-OICR ?

Reply:
See: https://github.com/reactome/data-release-pipeline/blob/ffb51885a7b2109bc1859e784673c0c345b6204e/downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/ProtegeExporter.java#L74

(also search for it here: https://github.com/reactome/data-release-pipeline/blob/ffb51885a7b2109bc1859e784673c0c345b6204e/downloadDirectory/README.md)

That option was added primarily for testing, since my local Perl installation needed to be added to the Perl "-I" includes explicitly. It could be used for other people for testing, or on the server if necessary Perl files are not in the correct place.

Reply:
Could a brief comment be included in the file to highlight this?